src/loaders/ccloudResourceLoader.test.ts

```diff
@@ -373,11 +383,10 @@ describe("CCloudResourceLoader", () => {
 
       // Test the args passed to the API.
       const args = flinkArtifactsApiStub.listArtifactV1FlinkArtifacts.getCall(0).args[0];
+      assert.ok(args, "Expected args to be defined");
       assert.strictEqual(args.cloud, TEST_CCLOUD_FLINK_COMPUTE_POOL.provider);
       assert.strictEqual(args.region, TEST_CCLOUD_FLINK_COMPUTE_POOL.region);
       assert.strictEqual(args.environment, TEST_CCLOUD_FLINK_COMPUTE_POOL.environmentId);
-      assert.strictEqual(args.page_size, 100);
-      assert.strictEqual(args.page_token, "");
```

Comment:
Ah sorry, I meant we could still keep these two assertions with `strictEqual` instead of switching them to the `assert.ok` use - let's add these back and then this branch will be good to go.